tests/conftest.py

```diff
@@ -135,6 +135,21 @@ def bipartite_graph4():
     return G
 
 
+@pytest.fixture
+def bipartite_dihypergraph1():
+    G = nx.DiGraph()
+    G.add_nodes_from([1, 2, 3, 4], bipartite=0)
+    G.add_nodes_from(["a", "b", "c"], bipartite=1)
+    G.add_edges_from([
```

Comment:
I switched the order of the edge pair so it works with `.edges`, otherwise we need to change to `.in_edges`